src/main/resources/jenkins/plugins/git/Messages.properties

```
@@ -25,3 +25,4 @@ GitSCMSource.DisplayName=Git
 GitStep.git=Git
 within.Repository=Within Repository
 additional=Additional
+GitUsernamePasswordBind.DisplayName=Git Username and Password
```

I don't see any mention of `GitUsernamePasswordBind` anywhere else in the code.  Shouldn't this be removed or replaced with a reference to the current class name?
```suggestion
GitUsernamePasswordBind.DisplayName=Git Username and Password
```